keyboards/preonic/keymaps/pezhore/keymap.c

```diff
@@ -0,0 +1,328 @@
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+
+enum preonic_layers {
+  _QWERTY,
+  _FN,
+  _NUMPAD,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum preonic_keycodes {
+  QWERTY = SAFE_RANGE,
+  FN,
+  NUMPAD,
+  LOWER,
+  RAISE,
+  SHRUG,  // ¯\_(ツ)_/¯
+  TFLIP,  // (╯°□°)╯︵ ┻━┻
+  POOP,   // 💩
+  DPOINT, // (ಠ_ಠ)
+  STRUT   // ᕕ( ᐛ )ᕗ
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+ * ,-----------------------------------------------------------------------------------.
+ * |   `  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Tab  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Del  |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | Esc  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | FN   | Ctrl | GUI  | ALT  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_QWERTY] = LAYOUT_preonic_grid(
+  KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
+  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_DEL,
+  KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+  KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,
+  FN,      KC_LCTL, KC_LGUI, KC_LALT, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
+),
+
+/* FN
+ * ,-----------------------------------------------------------------------------------.
+ * |RGBTgl|Brigt-|Brigt+|      |      |      |      |      |      |      |      | POOP |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | RGB+ | Hue- | Hue+ |      |      |      |      |      |      |      |PrtScr|UC_MOD|
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | RGB- | Sat- | Sat+ |      |      |      |      |      |      |      |      |UC_Wnc|
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |Effct-|Effct+|      |      |      |      |      |      |      |      |UC_Lin|
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |SHRUG |TFLIP |DPOINT|STRUT |      |      |      |QWERTY|NUMPAD|      |UC_Mac|
```

Comment:
Or however you want it phrased:

```suggestion
 * |      |SHRUG |TFLIP |DPOINT|STRUT |      |      |      |      |TgNmpd|      |UC_Mac|
```